keyboards/primekb/meridian/rules.mk

```
@@ -17,7 +17,8 @@ BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
 RGBLIGHT_ENABLE = yes       # Enable keyboard RGB underglow
 BLUETOOTH_ENABLE = no       # Enable Bluetooth
 AUDIO_ENABLE = no           # Audio output
-WS2812_DRIVER = spi
 
 # Enter lower-power sleep mode when on the ChibiOS idle thread
 OPT_DEFS += -DCORTEX_ENABLE_WFI_IDLE=TRUE
+
+DEFAULT_FOLDER = primekb/meridian/v1.1
```

Should be the old folder to maintain backwards compatibility
```suggestion
DEFAULT_FOLDER = primekb/meridian/v1
```

---

I chose 1.1 as the default since there about 5x as many 1.1 PCBs available than 1.0

Switch to bitbang for 2 reasons: 1) there seemed to be a bug introduced in the SPI driver after the initial Meridian commit, as reported here: https://github.com/qmk/qmk_firmware/issues/11755. And 2)  So far as I can tell I can't define data transfer times through the SPI driver (ie T0H, T1H, T0L, T1L), which is a must. 

@drashna said to use rgblight layers, so I changed it. Guys I don't care how it's implemented so long as it works and it gets approved/merged. Lets all get on the same page, tell me how you want this part done and I'll do it. 

Thanks guys